dts/README.md

```diff
@@ -31,7 +31,7 @@ URL:
    https://github.com/STMicroelectronics/STM32_open_pin_data
 
 Commit:
-   84963babb708a3465cf4913a83a9d7cb8e5ac13f
+   76cea512bdcafd6be002393400ebd925aec90531
```

Comment:
if using another version, first run the generator with the new version and then add LCD support.